README.md

```diff
@@ -6,17 +6,19 @@ This repository contains the source files for the kubescape.io website.
 
 ### Local testing
 
-We use [mkdocs](https://www.mkdocs.org/) with the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme. 
+We use [mkdocs](https://www.mkdocs.org/) with the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme to built the website. 
 
 To build and test locally: 
 
-* [install mkdocs-material](https://squidfunk.github.io/mkdocs-material/getting-started/) with `pip` or Docker 
-* run `mkdocs serve`
-
-## Local build
-
-Run `mkdocs build`, and the site will be built in the `site/` directory.
+* Ensure you have python and pip installed.
+* you can install all the required dependencies such as rrs plugin, mkdocs and more using the following command:
+```bash 
+pip install -r requirements.txt
+```
+* [install mkdocs-material](https://squidfunk.github.io/mkdocs-material/getting-started/) with `pip` or Docker.
+* Once you have all the dependencies installed, run `mkdocs build` and the site will be built in the `site/` directory.
+* After building the website locally, you can run the wesbite locally using the command `mkdocs serve`.
```

Comment:
````suggestion
* Ensure you have `python` and `pip` installed.
* Install the required dependencies:
```bash 
pip install -r requirements.txt
```
* Run `mkdocs serve`

## Local build

Run `mkdocs build`, and the site will be built in the `site/` directory.
````

Reply:
* running `pip` with the requirements file will install mkdocs-material, so we don't need to refer to that.
* Beware of "wesbite" vs "website".
* I suggest we keep testing and build instructions separate.

Reply:
Done, Resolved in new commit